Scala/Scala.sublime-syntax

```diff
@@ -7,6 +7,9 @@ file_extensions:
   - sbt
 scope: source.scala
 variables:
+  # lookahead/behind for operators
+  oplookbehind: '(?<=[[[:alpha:]]0-9\s\(\)\[\]\{\}])'
```

Comment:
I don't have time right ATM to look into the specifics of your implementation, but I do believe we can use a context push/pop to handle situations where you only want matches to happen if the preceding characters matched a certain pattern.

Here are some more examples (I linked to the JavaScript one in the main PR comments):

 - https://github.com/sublimehq/Packages/commit/dba0eaed9285d36590013ea5c5135fd2b90f9642
 - https://github.com/sublimehq/Packages/blob/master/Ruby/Ruby.sublime-syntax#L1042-L1061
 - https://github.com/sublimehq/Packages/blob/master/Ruby/Ruby.sublime-syntax#L316-L379